src/datatypes/interval.coffee

```diff
@@ -165,6 +165,11 @@ module.exports.Interval = class Interval
     else # ol is null
       null
 
+  sameAs: (other, precision) ->
+    if (@ == null or other == null) then return null
```

Comment:
This branch is not tested:
<img width="445" alt="image" src="https://user-images.githubusercontent.com/14879344/50228951-2e0b6100-0377-11e9-93e1-fb67ab6bf6d1.png">

Reply:
I had a test for that, but null values are handled earlier in the execution. I have removed that line.